packages/grid/helm/syft/templates/seaweedfs/seaweedfs-statefulset.yaml

```diff
@@ -37,8 +46,6 @@ spec:
                 secretKeyRef:
                     name: {{ .Values.seaweedfs.secretKeyName | required "seaweedfs.secretKeyName is required" }}
                     key: s3RootPassword
-            - name: MOUNT_API_PORT
```

Comment:
Q: Why are we removing this ?

Reply:
Not really required. Because (1) not exposed to the net and (2) the idea is to avoid dynamic port assignment for simplicity and follow standard port numbers for internal apis.